index.html

```diff
@@ -1,327 +1,339 @@
-    <title>Letaru Linda Masia</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
-      rel="stylesheet"
-    />
-  </head>
-  <body>
-    <header class="site-header">
-      <a href="#"> Letaru Linda Masia</a>
-      <nav class="nav-menu">
-        <img src="./images/Vector.png" alt="Vector image " />
-      </nav>
-    </header>
-    <main>
-      <section class="container">
-        <div class="text">
-          <h1>Hey There. I'm Linda</h1>
-          <h3>I'm a Software Developer.</h3>
-          <p>
-            I can help you build a product , feature or website Look through
-            some of my work and experience! If you like what you see and have a
-            project you need coded, don’t hestiate to contact me.
-          </p>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Linda Letaru Masia</title>
+</head>
+<body class="mainBody">
+<!-- Navbar -->
+<header class="navbar">
+  <a class="logo" href="#">Linda Letaru</a>
+  <div class="empty"></div>
+  <ul class="navbar-menu">
+    <li><a href="#">Portfolio</a></li>
+    <li><a href="#">About</a></li>
+    <li><a href="#">Contact</a></li>
+    <li>
+      <a href="#"><img src="images/inbox.svg" alt="inbox" /></a>
+    </li>
+  </ul>
+  <nav>
+    <img class="hand-burger" src="images/Union.svg" alt="Menu Icon" />
+  </nav>
+</header>
+
+<!-- Header Section -->
+<section class="main-body">
+  <div class="main-text">
+    <h1 class="main-heading">
+      Hey There.<br />
+      I'm Linda.
+    </h1>
+    <h2 class="sub-heading">I am a Software Developer</h2>
+    <p>
+      I can help you build a product , feature or website Look through some
+      of my work and experience! If you like what you see and have a project
+      you need coded, don't hestiate to contact me.
+    </p>
+  </div>
+
+  <div>
+    <ul class="icon">
+      <li>
+        <a href="#"><img class="social" src="images/GitHub.svg" alt="GitHub" /></a>
+      </li>
+      <li>
+        <a href="#"><img class="social" src="images/Linkdin.svg" alt="Linkdin" /></a>
+      </li>
+      <li>
+        <a href="#"><img class="social" src="images/Hand-Icon.svg" alt="HandIcon" /></a>
+      </li>
+      <li>
+        <a href="#"><img class="social" src="images/Twitter.svg" alt="Twitter" /></a>
+      </li>
+      <li>
+        <a href="#"><img class="social" src="images/M-Icon.svg" alt="M-Icon" /></a>
+      </li>
+    </ul>
+  </div>
+</section>
+
+<!-- Work Section -->
+<section class="work-section">
+  <div class="work-div">
+    <h1>My Recent Works</h1>
+    <div class="recent-work-underline">
+      <div class="break-line"></div>
+    </div>
+  </div>
+
```

Comment:
You seemed to have removed the `see project` button entirely, however, the previous reviewer only meant that it is expected by the design to appear on hover. I suggest that you stick to that.


![image](https://user-images.githubusercontent.com/20554583/187043734-1485e904-6155-4045-89e6-91210647147a.png)